tests/integration/testdata/sync/template.yaml

```diff
@@ -0,0 +1,70 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Globals:
+  Function:
+    Timeout: 3
+
+Resources:
+  HelloWorldFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: Python/function/
+      Handler: app.lambda_handler
+      Runtime: python3.7
+      Layers:
+        - Ref: HelloWorldLayer
+      Tracing: Active
+
+  HelloWorldApi:
+    Type: AWS::Serverless::Api
```

Comment:
Do we want to test HttpApi as well?

Reply:
Yes we definitely want to test HttpApi's, but currently I have not found a way that HttpApi's can be tested successfully with our integ tests. HttpApi's can only be tested by sending actual http requests. The one I commented out was something I attempted to test it but it doesn't return the expected message. If I can get a working test call I will add it.